WIP: improvements on docs (#83)
oeway/ImJoy: docs/development.md

The above `<config>` block will create a plugin with two tags(`Single` and `Multi`). During installation, the user which run-time behavior he would like to use (either a single instance of the plugin process (`Single`), or multiple plugin processes if multiple ImJoy interface with the same workspace are opened (`Multi`)).

### Virtual environments
By default, Python plugins from ImJoy will be executed in the default conda environment

> (e.g. Python 3.6). However, you these plugins can have specific virtual conda environment, which provides a way to isolate plugins. You can therefore run them with different versions of Python, or use different conda or pip packages.

Reviewer: ```suggestion
(e.g. Python 3.6). However, these plugins can have specific virtual conda environments, which provide a way to isolate plugins. You can therefore run them with different versions of Python, or use different conda or pip packages.
```

> We encourage the use one virtual environments per plugin and tag to guarantee stability.  We further recommend specifying the full version numbers (X.X.X) for python, pip and conda package. Like this, conda will try to reuse the packages with same version (and python version) across virtual environments, which reduces the required disk space. As an example, the following two environments will reuse the specified scipy package,

Reviewer: ```suggestion
We encourage the use of virtual environments per plugin and tag to guarantee stability.  We further recommend specifying the full version numbers (X.X.X) for python, pip and conda packages. By specifying the full version, conda will try to reuse the packages with same version (and python version) across virtual environments, which reduces the required disk space. As an example, the following two environments will reuse the specified scipy package,
```
